registry/handlers/hooks.go

```diff
@@ -0,0 +1,28 @@
+package handlers
+
+import (
+	"github.com/Sirupsen/logrus"
+)
+
+// LogHook is for hooking Panic in web application
+type LogHook struct {
+	LevelsParam []string
+	MailParam   Mail
```

Comment:
@RichardScothern , this struct (defined in mail.go) seems better to be independent to which in configuration. And configuration parameters should not be referred in many other packages.
it is what @stevvooe suggest to me.